src/Libraries/Microsoft.Extensions.Telemetry/Logging/LoggerEnrichmentOptions.cs

```diff
@@ -41,6 +41,14 @@ public class LoggerEnrichmentOptions
     /// </remarks>
     public bool UseFileInfoForStackTraces { get; set; }
 
+    /// <summary>
+    /// Gets or sets a value indicating whether to include exception messages in stack traces.
+    /// </summary>
+    /// <value>
+    /// This defaults to <see langword="false"/>.
```

Comment:
Worth mentioning that `CaptureStackTraces` should be enabled, otherwise this option doesn't change anything

Reply:
Done.